content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -257,6 +275,22 @@ fail validation.
 				</ul>
 			</td>
 		</tr>
+		<tr>
+			<td style="white-space: nowrap">Windows HostProcess</td>
```

Comment:
This looks like a duplacet of the `HostProcess` row added above. Am I missing something?

Reply:
Nope it's a duplicate, removing it - thanks!